scripts/InstallerYAMLSyntax.md

```diff
@@ -115,7 +115,7 @@ Item | Description | Default
 `httpsProxy` | HTTPS Proxy as a string | `-UNDEFINED-`
 `swupdMirror` | URL of the swupd stream to use. Useful for installing from a local mirror or from a locally published mix. | `-UNDEFINED-`
 `hostname` | Name of the host system | `-UNIQUE RANDOM-`
-`version` | Version of Clear Linux OS to installer | `-LATEST-`
+`version` | Version of Clear Linux OS to installer | `-VERSION_ON_BUILD_SYSTEM-`
```

Comment:
Can you also please fix the typo next to it. i.e "Change Version of Clear Linux OS to _installer_" to "Version of Clear Linux OS to _install_"

Reply:
Done.